site/index.html

```diff
@@ -1,25 +1,77 @@
 <!DOCTYPE html>
-<html>
-    <head>
-        <meta charset="utf-8">
-        <meta name="author" content="Juan Roy">
-        <title>CSD112</title>
-    </head>
-    <body>
-        <h1>Welcome to my (juanroy012) page</h1>
-        <p>This is my first project for introduction to web application.</p>
-        <img src="..\assets\lab-1\image.png" alt="Logo">
-        <h3>Example of ul (Unordered List)</h3>
-            <ul>
-                <li>ul means unordered list, which can be used to create list in non-specific order</li>
-                <li>use "li" inside of the body of the ul to add more content</li>
-                <li>This is my 3rd line inside of ul</li>
-            </ul>
-        <h3>Example of ol (Ordered list)</h3>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="author" content="Juan Roy">
+    <title>CSD112</title>
+    <link rel="icon" type="image/x-icon" href="../assets/Lab-2/favicon.ico">
```

Comment:
```suggestion
    <link rel="icon" type="image/x-icon" href="assets/Lab-2/favicon.ico">
```
If you want to use the assets directory it needs to live within the site directory